src/gt4py/backend/dawn_backends.py

```diff
@@ -139,6 +189,11 @@ def visit_Assign(self, node: gt_ir.Assign, **kwargs):
         stmt = sir_utils.make_assignment_stmt(left, right, "=")
         return stmt
 
+    def visit_AugAssign(self, node: gt_ir.AugAssign):
+        bin_op = gt_ir.BinOpExpr(lhs=node.target, op=node.op, rhs=node.value)
+        assign = gt_ir.Assign(target=node.target, value=bin_op)
+        return self.visit_Assign(assign)
```

Comment:
This should also work calling the general dispatcher method `self.visit` instead of calling directly the handler of `Assign` nodes, and it is more robust.